sass/components/_header.scss

```diff
@@ -118,18 +132,22 @@
   @include govuk-width-container;
   @include govuk-responsive-margin(3, 'bottom');
   @include govuk-responsive-padding(3, 'top');
-  @include govuk-responsive-padding(3, 'bottom');
-  width: 100%;
```

Comment:
this was incorrectly overriding the gov.uk width causing part of the bar to be hidden in smaller widths